EIPS/eip-7612.md

```diff
@@ -142,6 +134,17 @@ def set_storage(
     state._overlay_tree.set(get_tree_key_for_storage_slot(addr, slot), value)
 ```
 
+Add the following function which is used when storing a contract in the tree:
+
+```python3
+def state_set_codechunk(state: State, addr: Address, chunk_num: int, chunk: Bytes):
+    state._overlay_tree.set(get_tree_key_for_code_chunk(addr, chunk_num), chunk)
+```
```

Comment:
this isn't used anywhere, so imo we should remove it unless it is used

Reply:
It's not referenced in this EIP, but it's referenced in another one which has this one as a dependency, [see here](https://github.com/ethereum/EIPs/pull/8752/files#diff-0476f5dcc1f4d253ea73e86a58163a7e194a1bfdb5464fe76e2f1cf97a3afc75R209).

Reply:
I would then put it the referring eip, but I will let the allmighty editors decide on this one. @g11tech ?

Reply:
Actually this code should be part of 6800, but that EIP doesn't have this "Python-style". But yep, let's see what @g11tech says and I can remove/move/add as needed.